pull_request_template.md

```diff
@@ -1,16 +0,0 @@
-<!--

```

Comment:
Now this file is deleted, but shouldn't be.

Reply:
Sry about that... :-(